joomla/joomla-cms
# [4.0] Align content info block icons

Pull Request for Issue # .

### Summary of Changes
makes all info-block icons fixed with so things line up.


### Testing Instructions
install "blog" sample data.
go to administrator/index.php?option=com_menus&view=item&client_id=0&layout=edit&id=103

change red options to match as shown.
![image](https://user-images.githubusercontent.com/1850089/91935553-961cea80-ecb3-11ea-8f6f-04d56714766c.png)

go to "about" page in front end..
![image](https://user-images.githubusercontent.com/1850089/91935616-b8166d00-ecb3-11ea-9208-c32e0ef3b413.png)

verify icons show.
![image](https://user-images.githubusercontent.com/1850089/91935655-cb293d00-ecb3-11ea-8893-873b1334f743.png)

verify icons & text line up.
![image](https://user-images.githubusercontent.com/1850089/91935749-06c40700-ecb4-11ea-8bbe-fc61f59fe0d8.png)

inspect icons to be sure the icon class ends in "fa-fw"
![image](https://user-images.githubusercontent.com/1850089/91935689-df6d3a00-ecb3-11ea-9a52-4da186faa0c6.png)


### Actual result BEFORE applying this Pull Request
Content & Icons didn't line up
![image](https://user-images.githubusercontent.com/1850089/91935896-51de1a00-ecb4-11ea-9914-598dc3020ae5.png)

### Expected result AFTER applying this Pull Request
everything is visually lined up.

### Documentation Changes Required

none
